api/registry.go

```diff
@@ -106,3 +107,18 @@ func RegisterBasicAPI(router gin.IRouter) {
 	router.GET("/", endpoint.GetStatus)
 	router.GET("/metrics", ginprom.PromHandler(promhttp.Handler()))
 }
+
+func RegisterLendingAPI(router gin.IRouter) {
+	router.GET("/v1/lending/providers", middleware.CacheMiddleware(time.Hour*10, func(c *gin.Context) {
+		endpoint.HandleLendingProviders(c, platform.LendingAPIs)
+	}))
+	router.GET("/v1/lending/assets/:provider", middleware.CacheMiddleware(time.Hour*10, func(c *gin.Context) {
+		endpoint.HandleLendingAssets(c, platform.LendingAPIs)
+	}))
+	router.GET("/v1/lending/assets/:provider/:asset", middleware.CacheMiddleware(time.Hour*10, func(c *gin.Context) {
```

Comment:
Why not delete `/v1/lending/assets/:provider/:asset` ? if you can use `v1/lending/assets/:provider`

There is no reason to have separate endpoint, if you have only a few assets

Reply:
I just tried to be friendly with the API users.  But agree, Keep It Simple Smart(tm)